pallets/liquidity-pools/src/lib.rs

```diff
@@ -193,6 +193,7 @@ pub mod pallet {
 			BalanceRatio = Self::BalanceRatio,
 			PoolId = Self::PoolId,
 			TrancheId = Self::TrancheId,
+			Moment = Seconds,
```

Comment:
Maybe it can be handled also by creating a `MomentOf<T>` type.

Having it as `Seconds` type (which is an `u64`) could lead to mismatching with other time precisions as milliseconds. But having it through the associated type can not.

Reply:
In the future Seconds and Milliseconds should have wrappers arround `u64` with math support to avoid mixing them

Reply:
But not super strong opinion, I have `pallet_loans` populated with `Seconds` right now 😆

Reply:
yeah we definitly should have strong typing around those two different units of time 👍 

is your suggestion adding a `Moment` type to the `Config` of liquidity pools and bound the `PriceValue<Moment = T::Moment>`? I want to keep the type at the message level concrete and not abstract since messages are type-fixed or else they will break; but we can have a `Into<u64>` or `Into<Seconds>` trait bound for this `Moment` type.

what do you think?

Reply:
> I want to keep the type at the message level concrete and not abstract since messages are type-fixed or else they will break

But yes, if you want to have more control over the type used, `T::moment` will work for sure!

Reply:
yeah exactly, I didn't want to do that because we then need to bound it to u64 to match the expected type in the message. Of course than in a normal scenario that's not what we want so that the pallet is actually generic over these types but for liquidity-pools, a multi-chain system, being type generic over the messages' fields is not desired.

So we can:

1. Keep thing as are with a strict bound on `PriceValue<Moment =` which is the least invasive

2. Add a `Moment` type to the pallet' `Config` and have `PriceValue<Moment ` bound to this type AND have the type implement `ToSeconds` or `Into<u64>`.

I am cool with the second but maybe we could keep things as are now and implement the latter change once we introduce strong typing for `Seconds` and `Millis`. wdyt @lemunozm ?

Reply:
Yeah, being the message who dictates here, I think leaving this as they're now is ok 👍🏻 
As you say, the mixing issue will be fixed with strong typing in `Seconds`.

Reply:
Hit the merge button! 🚀